cca_zoo/test/test_model_selection.py

```diff
@@ -0,0 +1,12 @@
+from cca_zoo.model_selection import permutation_test_score
```

Comment:
```
from cca_zoo.model_selection import permutation_test_score
from sklearn.utils.validation import check_random_state
from cca_zoo.models import CCA
from cca_zoo.data import generate_covariance_data

n = 50
rng = check_random_state(0)

# this generates data with ground truth number of latent dimensions
(X, Y), (_, _) = generate_covariance_data(1000, [5, 5], latent_dims=4, correlation=[1, 1, 0, 0], random_state=rng)


def test_perm_test():
    p, A, B, U, V = permutation_test_score(X=X, Y=Y, estimator=CCA, latent_dims=4, random_state=rng)
    assert p[0] < 0.05
    assert p[1] < 0.05
    assert p[2] > 0.05
    assert p[3] > 0.05
```

Wonder if something like this might make a good test.